diff --git a/node_modules/filehound/index.d.ts b/node_modules/filehound/index.d.ts
index c3af29d..3507dc2 100644
--- a/node_modules/filehound/index.d.ts
+++ b/node_modules/filehound/index.d.ts
@@ -105,7 +105,7 @@ declare module 'filehound' {
      *  .find()
      *  .each(console.log);
      */
-    public addFilter(filter: (...any) => boolean): FileHound;
+    public addFilter(filter: (...args: any[]) => boolean): FileHound;
 
     /**
      * Defines the search paths
